ai_guidelines.md

```diff
@@ -0,0 +1,140 @@
+# Guidelines for CommCare HQ
+
+This document outlines recommended practices for working with the CommCare HQ
+codebase. These guidelines apply in the following circumstances:
+
+- Reviewing code or suggesting changes
+- Writing code and making changes
+- Assisting with version control
+
+## General Principles
+
+- Focus on clarity, maintainability, and performance
+- Consider both immediate needs and long-term implications
+
+## Communication
+
+### For the AI Assistant
+
+- The AI Assistant should maintain professional, focused communication.
+- The AI Assistant should ask clarifying questions whenever details are unclear,
+  or choices are not obvious.
+- When appropriate, the AI Assistant should make recommendations on how the
+  Developer can improve their prompt to get a more effective response.
+
+### For the Developer
+
+- The Developer should provide feedback to the AI Assistant to improve future
+  responses.
+- The Developer should update this document to assist the AI Assistant across
+  sessions, and to improve the interactions of other Developers.
+
+## Tech Stack
+
+- Backend: Python 3.9, Django 4.2
```

Comment:
It presents a maintenance burden to have hard-coded versions here. Would it work to instead reference pyproject.toml and uv.lock as the source of truth for version information?

Or maybe just remove the versions? None of the components below except for JavaScript have version numbers.

Reply:
862762e4a0f